docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -494,6 +495,34 @@ The `tableName` key defines the name of the join table. It has to be specified o
 
 :::::
 
+#### Custom fields
+
+[Custom fields](/developer-docs/latest/development/custom-field.md) extend Strapi’s capabilities by adding new types of fields to content-types. Custom fields are explicitly defined in the [attributes](#model-attributes) of a model with `type: customField`.
+Custom fields attributes also accept:
+
+- the `customField` additional parameter to mention the registered custom field to use, using:
+  - either the `plugin::plugin-name.fieldname` format for a custom field created by a plugin
+  - or the `global::api-name.fieldname` for a custom field specific to the current Strapi application
```

Comment:
```suggestion
  - or the `global::fieldname` for a custom field specific to the current Strapi application
```

Reply:
I agree with the suggested change, but I'm not sure if I'm supposed to commit the suggestion?

Reply:
Thank you, Rémi. No worries, and sorry for the confusion ^^ I'll handle the writing part on this PR. Just wanted to have your final check before committing 🤗